tools/fuzzer/main.c

```diff
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: BSD-3-Clause
+// SPDX-License-Identifier: BSD-4-Clause
```

Comment:
Why the licensing change? Sorry if it was already explained but it feels weird to me.

Reply:
According to https://spdx.org/licenses/BSD-4-Clause.html this license is actually rescinded (I think that means obsolete?)

Reply:
Seems like an unintentional typo, doesn't exist on original PR